source/jormungandr/jormungandr/interfaces/v1/serializer/pt.py

```diff
@@ -346,6 +348,29 @@ class CommercialModeSerializer(PbGenericSerializer):
     pass
 
 
+class CauseSerializer(PbNestedSerializer):
+    label = base.PbStrField(display_none=True, required=False)
+
+
+class EffectSerializer(PbNestedSerializer):
+    label = base.PbStrField(display_none=True, required=False)
+
+
+class CurrentAvailabilitySerializer(PbNestedSerializer):
+    status = EnumField(pb_type=CurrentAvailability.EquipmentStatus, display_none=False, required=False)
+    periods = PeriodSerializer(many=True, display_none=False, required=False)
+    updated_at = base.PbStrField(display_none=True, required=False)
```

Comment:
It's a bit weird to display `update_at` when none (especially that it's the only `display_none=True` from the object).

Reply:
Obviously, Copy-Paste gone wrong!!